src/plotman/configuration.py

```diff
@@ -35,6 +35,13 @@ def get_validated_configs():
     except marshmallow.exceptions.ValidationError as e:
         raise ConfigurationException(f"Config file at: '{config_file_path}' is malformed") from e
 
+def get_dst_directories(dir_cfg):
+    """Returns either (True, <Directories.dst>) or (False, <Directories.tmp>). If Directories.dst is None,
+    Use Directories.tmp as dst directory."""
+    if dir_cfg.dst is None:
+        (False, dir_cfg.tmp)
+    else:
+        (True, dir_cfg.dst)
```

Comment:
What do you think of making this a method or two on `Directories`?  Maybe `dir_cfg.dst_is_tmp()` and `dir_cfg.effective_dst_dirs()` or such?

Also, as is, this is missing returns for the specified tuples.